### PR TITLE
clusterRegistry interactions decreased in _updateRewards

### PR DESCRIPTION
**Pull request by prateekreddy**

### CODE REVIEW

File: contracts/Stake/StakeManager.sol [hunk 1]
```diff
@@ -5,8 +5,7 @@ import "@openzeppelin/contracts-ethereum-package/contracts/token/ERC20/ERC20.sol
 import "@openzeppelin/contracts-ethereum-package/contracts/ownership/Ownable.sol";
 import "@openzeppelin/contracts-ethereum-package/contracts/math/SafeMath.sol";
 import "./IRewardDelegators.sol";
-import "../governance/MPondLogic.sol";
-import "./IClusterRegistry.sol";
+import "../governance/mPondLogic.sol";
```
Comment (roshanrags) [inline contracts/Stake/StakeManager.sol]: seems wrong?

File: contracts/Stake/StakeManager.sol [hunk 2]
```diff
@@ -33,7 +32,6 @@ contract StakeManager is Initializable, Ownable {
     mapping(bytes32 => Token) tokenAddresses;
     MPondLogic MPOND;
     MPondLogic prevMPOND;
-    IClusterRegistry clusterRegistry;
```
Comment (roshanrags) [inline contracts/Stake/StakeManager.sol:36]: Storage change, should leave empty stub?

Comment (prateekreddy) [reply contracts/Stake/StakeManager.sol:36]: right, will change this

File: contracts/Stake/StakeManager.sol (hunk 1 above)
Comment (prateekreddy) [reply contracts/Stake/StakeManager.sol]: Ah... typo. Will look into it. It was compiling though.

**Pull request merged**